docs/basic-features/data-fetching.md

```diff
@@ -54,11 +54,14 @@ The `context` parameter is an object containing the following keys:
 - `params` contains the route parameters for pages using dynamic routes. For example, if the page name is `[id].js` , then `params` will look like `{ id: ... }`. To learn more, take a look at the [Dynamic Routing documentation](/docs/routing/dynamic-routes.md). You should use this together with `getStaticPaths`, which we’ll explain later.
 - `preview` is `true` if the page is in the preview mode and `undefined` otherwise. See the [Preview Mode documentation](/docs/advanced-features/preview-mode.md).
 - `previewData` contains the preview data set by `setPreviewData`. See the [Preview Mode documentation](/docs/advanced-features/preview-mode.md).
+- `locale` contains the active locale (if enabled)
+- `locales` contains all supported locales (if enabled)
 
 `getStaticProps` should return an object with:
 
 - `props` - A **required** object with the props that will be received by the page component. It should be a [serializable object](https://en.wikipedia.org/wiki/Serialization)
 - `revalidate` - An **optional** amount in seconds after which a page re-generation can occur. More on [Incremental Static Regeneration](#incremental-static-regeneration)
+- `notFound` - An **optional** boolean value specifying the page should handle as the 404 page returning the 404 status. More on [Incremental Static Regeneration](#incremental-static-regeneration)
```

Comment:
This sentence could be clarified.

Reply:
Tried tweaking it a bit, does that clarify it more or should we rephrase some more?

Reply:
Maybe  "an optional boolean value to allow the page to return a 404 status and page"